app/controllers/financials.js

```diff
@@ -0,0 +1,125 @@
+"use strict";
+
+const fs = require("fs"),
+  request = require("request"),
+  config = require("../../config/config"),
+  fileSystem = require("../helpers/file-system"),
+  EventEmitter = require("events").EventEmitter,
+  util = require("util");
+
+
+const FinancialsController = function(url, riseDisplayNetworkII, logger) {
+  EventEmitter.call(this);
+
+  this.url = url;
+  this.fileName = fileSystem.getFileName(this.url);
+  this.pathInCache = fileSystem.getPathInCache(this.url);
+  this.pathInDownload = fileSystem.getPathInDownload(this.url);
+  this.riseDisplayNetworkII = riseDisplayNetworkII;
+  this.logger = logger;
+};
+
+util.inherits(FinancialsController, EventEmitter);
+
+FinancialsController.prototype.getData = function(opts) {
+  let options = {};
+
+  if ( this.url ) {
+    options.url = this.url;
+
+    options.headers = {
+      "User-Agent": "request"
+    };
+    options.proxy = (this.riseDisplayNetworkII) ?
+      (this.riseDisplayNetworkII.get("activeproxy") || this.riseDisplayNetworkII.get("rcosproxy")) :
+      null;
+    options.timeout = config.requestTimeout;
+
+    if (opts) {
+      Object.assign(options.headers, opts);
+    }
+
+    fileSystem.getAvailableSpace(this.logger, (spaceInDisk)=>{
+      request(options, (error, res, body) => {
```

Comment:
@rodrigopavezi The `method` option already defaults to "GET" - https://goo.gl/gKYHru